src/main/bash/envs/kobman-KOBman.sh

```diff
@@ -3,23 +3,24 @@
 function __kobman_install_KOBman
 {
 	local environment_name="$1"
+	local dev_area_name="Dev_$environment_name"
```

Comment:
The user or applications may not follow a naming convention like Dev_ when it comes to selecting a name for the root folder where the environment should be hosted. So best to avoid Dev_ prefix .You will have to pass the argument or set a export variable with the default environment root folder i.e KOBMAN_ENV_ROOT.

Reply:
@panickervinod So, the user should give the name for their root folder?